### PR TITLE
refactor: performance tests

### PR DESCRIPTION
**Pull request by Mirko-von-Leipzig**
Simplification and renaming of performance tests, and updated results to our target versions.

### CODE REVIEW

File: src/tests/performance/mod.rs [hunk 1]
```diff
@@ -88,3 +86,16 @@ pub fn table_float_display(x: &f64) -> String {
 pub fn duration_as_ms(duration: Duration) -> f64 {
     duration.as_millis() as f64
 }
+
+/// Formats a [Table] with our style:
+///  - [pseudo style](Style) (todo - fix this link)
```
Comment (Mirko-von-Leipzig) [inline src/tests/performance/mod.rs:91]: I want this to link to [Style::pseudo()](https://docs.rs/tabled/0.2.1/tabled/style/struct.Style.html#method.pseudo), but I don't know how to link to an external crate's function?

Comment (niklaslong) [reply src/tests/performance/mod.rs:91]: Should be just [pseudo style](https://docs.rs/tabled/0.2.1/tabled/style/struct.Style.html#method.pseudo) no?

File: src/tests/performance/ping_pong.rs [hunk 2]
```diff
@@ -114,95 +123,85 @@ async fn throughput() {
...
         // add stats to table display
         table.add_row(RequestStats::new(
-            peers as u16,
+            synth_count as u16,
             PINGS,
             latencies,
             time_taken_secs,
         ));
+
+        println!("{}", table);
```
Comment (niklaslong) [inline src/tests/performance/ping_pong.rs:166]: Suggestion: only print the table at the end of the run?

Comment (Mirko-von-Leipzig) [reply src/tests/performance/ping_pong.rs:166]: Debug aid left behind :D will remove.

File: src/tests/performance/mod.rs (hunk 1 above)
Comment (Mirko-von-Leipzig) [reply src/tests/performance/mod.rs:91]: That works; but I was hoping for something a bit more automated? (i.e. link to the version we are actually running). What bothers me is that `(Style)` works just fine, but I can't get any further.

Comment (niklaslong) [reply src/tests/performance/mod.rs:91]: Oh you mean the method? What about `[...](Struct::pseudo)`?

**Review by niklaslong — APPROVED** [review]

**Pull request merged**